packages/forms/src/UIForm/fields/RadioOrSelect/RadioOrSelect.component.js

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes } from 'react';
+import Radio from '../Radios';
+import Select from '../Select';
+
+const MAX_TO_RADIO = 2;
```

Comment:
In fact I asked on the Slack forms channel why this component existed, because it is based on static enum.
I don't see the benefit the set a 'RadioOrSelect' with static enum, I would have set Radio or Select widget directly.
Adding this to the api is even worst, it means the dev knows the number of possible value, et set a threshold to use one of the other, instead of using it directly.

I added it to ensure compatibility.